tests/sentry/tasks/test_sentry_apps.py

```diff
@@ -313,6 +313,59 @@ def test_integration_org_error_created_doesnt_send_webhook(self, safe_urlopen):
         assert not safe_urlopen.called
 
 
+class TestSendResourceChangeWebhook(TestCase):
+    def setUp(self):
+        self.project = self.create_project()
+        self.sentry_app_1 = self.create_sentry_app(
+            organization=self.project.organization,
+            events=["issue.created"],
+            webhook_url="https://google.com",
+        )
+        self.install_1 = self.create_sentry_app_installation(
+            organization=self.project.organization, slug=self.sentry_app_1.slug
+        )
+        self.sentry_app_2 = self.create_sentry_app(
+            organization=self.project.organization,
+            events=["issue.created"],
+            webhook_url="https://apple.com",
+        )
+        self.install_2 = self.create_sentry_app_installation(
+            organization=self.project.organization,
+            slug=self.sentry_app_2.slug,
+            status=SentryAppInstallationStatus.INSTALLED,
```

Comment:
nit: I think `INSTALLED` is the default so you don't actually need to pass it in